app/services/user_api_services.rb

```diff
@@ -0,0 +1,44 @@
+class UserApiServices
+  def self.get_users_books(auth_token, filter = nil)
+    response = conn.get('api/v1/user/books') do |req|
+      req.params[:filter] = filter
+      req.params[:auth_token] = auth_token
+    end
+    
+    JSON.parse(response.body, symbolize_names: true)
+  end
+
+  def self.add_a_book(volume_id, auth_token, shelf_id = 2)
+    conn.post "/api/v1/user/books" do
+      req.params[:volume_id] = volume_id
+      req.params[:auth_token] = auth_token
+      req.params[:shelf_id] = shelf_id
+    end
+    response.status
+  end
+
+  def self.remove_a_book(volume_id, auth_token, shelf_id)
+    conn.delete "/api/v1/user/books" do
```

Comment:
this endpoint has been updated to be more restful - you'll need to append the volume_id to the url now: 

Endpoint: delete: 'https://epeolatry-back-end.herokuapp.com/api/v1/user/books/:volume_id’
Params: { auth_token: auth_token_for_user, shelf_id: ‘2, 3, or 4’ }

Reply:
And added, Thank you!